data/toc.yaml

```diff
@@ -2053,6 +2051,15 @@ Manuals:
   - path: /docker-hub/release-notes/
     title: Release notes
 
+- sectiontitle: Docker account
+  section:
+  - path: /docker-account/
```

Comment:
how about `accounts` for the section slug? i.e. `https//docs.docker.com/accounts/manage-account`